includes/managers/class-fs-admin-menu-manager.php

```diff
@@ -220,6 +232,16 @@ function has_custom_parent() {
 		}
 
 		/**
+		 * @author Leo Fajardo (@leorw)
+		 * @since  1.2.0
+		 *
+		 * @return bool
+		 */
+		function menu_exists() {
```

Comment:
@fajardoleo when it's boolean methods, it's better to call it `is_menu_exists()` or `has_menu()`. Pick your favorite. It's also relevant for the private member - `$this->_menu_exists`